--- dokuwiki-2013-03-05/./lib/plugins/authad/auth.php~	2013-03-03 21:16:43.000000000 +0200
+++ dokuwiki-2013-03-05/./lib/plugins/authad/auth.php	2013-03-05 12:41:35.995501610 +0200
@@ -2,7 +2,7 @@
 // must be run within Dokuwiki
 if(!defined('DOKU_INC')) die();
 
-require_once(DOKU_PLUGIN.'authad/adLDAP/adLDAP.php');
+require_once '/usr/share/php/adLDAP.php';
 
 /**
  * Active Directory authentication backend for DokuWiki
