--- dokuwiki-2018-04-22a/vendor/composer/autoload_real.php~	2018-05-03 09:36:13.000000000 +0300
+++ dokuwiki-2018-04-22a/vendor/composer/autoload_real.php	2018-11-10 19:37:33.858380525 +0200
@@ -23,7 +23,7 @@
         self::$loader = $loader = new \Composer\Autoload\ClassLoader();
         spl_autoload_unregister(array('ComposerAutoloaderInita19a915ee98347a0c787119619d2ff9b', 'loadClassLoader'));
 
-        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION') && (!function_exists('zend_loader_file_encoded') || !zend_loader_file_encoded());
+        $useStaticLoader = false; // autoload_static.php patching is annoying and not worth the effort
         if ($useStaticLoader) {
             require_once __DIR__ . '/autoload_static.php';
 
