diff -ur dokuwiki-rc2008-03-31/inc/init.php dokuwiki-rc2008-03-31.new/inc/init.php
--- dokuwiki-rc2008-03-31/inc/init.php	2008-03-31 20:11:17.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/inc/init.php	2008-04-01 09:44:54.001374243 +0200
@@ -14,7 +14,7 @@
   if(!defined('DOKU_INC')) define('DOKU_INC',fullpath(dirname(__FILE__).'/../').'/');
 
   // define config path (packagers may want to change this to /etc/dokuwiki/)
-  if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
+  if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'/etc/webapps/dokuwiki/');
 
   // check for error reporting override or set error reporting to sane values
   if (!defined('DOKU_E_LEVEL') && @file_exists(DOKU_CONF.'report_e_all')) {
Tylko w dokuwiki-rc2008-03-31.new/inc: init.php~
diff -ur dokuwiki-rc2008-03-31/inc/plugin.php dokuwiki-rc2008-03-31.new/inc/plugin.php
--- dokuwiki-rc2008-03-31/inc/plugin.php	2008-03-31 20:11:18.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/inc/plugin.php	2008-04-01 09:46:37.316313286 +0200
@@ -146,7 +146,7 @@
    */
   function readDefaultSettings() {
 
-    $path = DOKU_PLUGIN.$this->getPluginName().'/conf/';
+    $path = DOKU_PLUGIN.$this->getPluginName().'/etc/webapps/dokuwiki/';
     $conf = array();
 
     if (@file_exists($path.'default.php')) {
Tylko w dokuwiki-rc2008-03-31.new/inc: plugin.php~
diff -ur dokuwiki-rc2008-03-31/inc/template.php dokuwiki-rc2008-03-31.new/inc/template.php
--- dokuwiki-rc2008-03-31/inc/template.php	2008-03-31 20:11:18.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/inc/template.php	2008-04-01 09:46:11.975101828 +0200
@@ -1081,7 +1081,7 @@
  */
 function tpl_loadConfig(){
 
-  $file = DOKU_TPLINC.'/conf/default.php';
+  $file = DOKU_TPLINC.'/etc/webapps/dokuwiki/default.php';
   $conf = array();
 
   if (!@file_exists($file)) return false;
Tylko w dokuwiki-rc2008-03-31.new/inc: template.php~
diff -ur dokuwiki-rc2008-03-31/install.php dokuwiki-rc2008-03-31.new/install.php
--- dokuwiki-rc2008-03-31/install.php	2008-03-31 20:11:18.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/install.php	2008-04-01 09:44:43.367532550 +0200
@@ -6,8 +6,8 @@
  */
 
 if(!defined('DOKU_INC')) define('DOKU_INC',dirname(__FILE__).'/');
-if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
-if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_INC.'conf/');
+if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'/etc/webapps/dokuwiki/');
+if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_INC.'/etc/webapps/dokuwiki/');
 
 // check for error reporting override or set error reporting to sane values
 if (!defined('DOKU_E_LEVEL')) { error_reporting(E_ALL ^ E_NOTICE); }
@@ -389,15 +389,15 @@
 
     $dirs = array(
         'conf'      => DOKU_LOCAL,
-        'data'      => DOKU_INC.'data',
-        'pages'     => DOKU_INC.'data/pages',
-        'attic'     => DOKU_INC.'data/attic',
-        'media'     => DOKU_INC.'data/media',
-        'meta'      => DOKU_INC.'data/meta',
-        'cache'     => DOKU_INC.'data/cache',
-        'locks'     => DOKU_INC.'data/locks',
-        'index'     => DOKU_INC.'data/index',
-        'tmp'       => DOKU_INC.'data/tmp'
+        'data'      => DOKU_INC,
+        'pages'     => DOKU_INC.'/pages',
+        'attic'     => DOKU_INC.'/attic',
+        'media'     => DOKU_INC.'/media',
+        'meta'      => DOKU_INC.'/meta',
+        'cache'     => DOKU_INC.'/cache',
+        'locks'     => DOKU_INC.'/locks',
+        'index'     => DOKU_INC.'/index',
+        'tmp'       => DOKU_INC.'/tmp'
     );
 
     $ok = true;
Tylko w dokuwiki-rc2008-03-31.new/: install.php~
diff -ur dokuwiki-rc2008-03-31/lib/plugins/config/settings/config.class.php dokuwiki-rc2008-03-31.new/lib/plugins/config/settings/config.class.php
--- dokuwiki-rc2008-03-31/lib/plugins/config/settings/config.class.php	2008-03-31 20:11:18.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/lib/plugins/config/settings/config.class.php	2008-04-01 09:47:59.423571814 +0200
@@ -224,8 +224,8 @@
      * load metadata for plugin and template settings
      */
     function get_plugintpl_metadata($tpl){
-      $file     = '/conf/metadata.php';
-      $class    = '/conf/settings.class.php';
+      $file     = '/etc/webapps/dokuwiki/metadata.php';
+      $class    = '/etc/webapps/dokuwiki/settings.class.php';
       $metadata = array();
 
       if ($dh = opendir(DOKU_PLUGIN)) {
@@ -270,7 +270,7 @@
      * load default settings for plugins and templates
      */
     function get_plugintpl_default($tpl){
-      $file    = '/conf/default.php';
+      $file    = '/etc/webapps/dokuwiki/default.php';
       $default = array();
 
       if ($dh = opendir(DOKU_PLUGIN)) {
Tylko w dokuwiki-rc2008-03-31.new/lib/plugins/config/settings: config.class.php~
diff -ur dokuwiki-rc2008-03-31/lib/plugins/syntax.php dokuwiki-rc2008-03-31.new/lib/plugins/syntax.php
--- dokuwiki-rc2008-03-31/lib/plugins/syntax.php	2008-03-31 20:11:19.000000000 +0200
+++ dokuwiki-rc2008-03-31.new/lib/plugins/syntax.php	2008-04-01 09:46:58.670667479 +0200
@@ -256,7 +256,7 @@
    */
   function readDefaultSettings() {
 
-    $path = DOKU_PLUGIN.$this->getPluginName().'/conf/';
+    $path = DOKU_PLUGIN.$this->getPluginName().'/etc/webapps/dokuwiki/';
     $conf = array();
 
     if (@file_exists($path.'default.php')) {
Tylko w dokuwiki-rc2008-03-31.new/lib/plugins: syntax.php~
