--- dokuwiki-2018-04-22c.org/inc/common.php	2022-09-27 22:46:33.691324681 +0200
+++ dokuwiki-2018-04-22c/inc/common.php	2022-09-27 22:46:33.924670225 +0200
@@ -897,6 +897,34 @@ function gethostsbyaddrs($ips) {
     }
 }
 
+
+/**
+ * Returns notify address for page $id
+ *
+ * @param  string  $id       The changed page
+ * @param  mixed   $notify   The address (string) or rules where to notify
+ *
+ * @author Elan Ruusamäe <glen@delfi.ee>
+ */
+function notifyNS($id, $notify) {
+  if (empty($notify)) {
+    return '';
+  }
+
+  if (is_string($notify)) {
+    return $notify;
+  }
+
+  // process $notify for pages
+  foreach ($notify as $rule => $address) {
+    if (preg_match('/'.$rule.'/ui',':'.$id)) {
+      return $address;
+    }
+  }
+
+  return '';
+}
+
 /**
  * Checks if a given page is currently locked.
  *
@@ -1435,10 +1463,10 @@ function notify($id, $who, $rev = '', $s
 
     // decide if there is something to do, eg. whom to mail
     if($who == 'admin') {
-        if(empty($conf['notify'])) return false; //notify enabled?
         if($conf['useacl'] && $_SERVER['REMOTE_USER'] && $minor) return; //skip minors
         $tpl = 'mailtext';
-        $to  = $conf['notify'];
+        $to  = notifyNS($id, $conf['notify']);
+        if(empty($to)) return; //notify enabled?
     } elseif($who == 'subscribers') {
         if(!actionOK('subscribe')) return false; //subscribers enabled?
         if($conf['useacl'] && $INPUT->server->str('REMOTE_USER') && $minor) return false; //skip minors
--- dokuwiki-2018-04-22c.org/inc/media.php	2020-05-01 12:38:20.000000000 +0200
+++ dokuwiki-2018-04-22c/inc/media.php	2022-09-27 22:46:33.924670225 +0200
@@ -647,10 +647,11 @@ function media_contentcheck($file,$mime)
  */
 function media_notify($id,$file,$mime,$old_rev=false){
     global $conf;
-    if(empty($conf['notify'])) return false; //notify enabled?
+    $to = notifyNS($id, $conf['notify']);
+    if(empty($to)) return false; //notify enabled?
 
     $subscription = new Subscription();
-    return $subscription->send_media_diff($conf['notify'], 'uploadmail', $id, $old_rev);
+    return $subscription->send_media_diff($to, 'uploadmail', $id, $old_rev);
 }
 
 /**
--- dokuwiki-2018-04-22c.org/inc/subscription.php	2020-05-01 12:38:20.000000000 +0200
+++ dokuwiki-2018-04-22c/inc/subscription.php	2022-09-27 22:46:33.924670225 +0200
@@ -489,7 +489,8 @@ class Subscription {
      */
     public function send_register($login, $fullname, $email) {
         global $conf;
-        if(empty($conf['registernotify'])) return false;
+        $to = notifyNS($id, $conf['registernotify']);
+        if(empty($to)) return false;
 
         $trep = array(
             'NEWUSER' => $login,
@@ -498,7 +499,7 @@ class Subscription {
         );
 
         return $this->send(
-            $conf['registernotify'],
+            $to,
             'new_user',
             $login,
             'registermail',
@@ -682,7 +683,8 @@ class Subscription {
 
                 $level = auth_aclcheck($id, $user, $userinfo['grps']);
                 if($level >= AUTH_READ) {
-                    if(strcasecmp($userinfo['mail'], $conf['notify']) != 0) { //skip user who get notified elsewhere
+                    $to = notifyNS($id, $conf['notify']);
+                    if(strcasecmp($userinfo['mail'], $to) != 0) { //skip user who get notified elsewhere
                         $result[$user] = $userinfo['mail'];
                     }
                 }
