--- dokuwiki-2009-02-14/inc/parser/xhtml.php~	2009-02-14 14:13:25.000000000 +0200
+++ dokuwiki-2009-02-14/inc/parser/xhtml.php	2009-03-12 22:05:27.933270570 +0200
@@ -680,6 +680,9 @@
         list($ext,$mime,$dl) = mimetype($src);
         if(substr($mime,0,5) == 'image' && $render){
             $link['url'] = ml($src,array('id'=>$ID,'cache'=>$cache),($linking=='direct'));
+            if ($linking != 'direct' && $linking != 'nolink') {
+                $link['more'] .= 'directlink="'.ml($src,array('id'=>$ID,'cache'=>$cache), true). '"';
+            }
         }elseif($mime == 'application/x-shockwave-flash' && $render){
             // don't link flash movies
             $noLink = true;
