--- dokuwiki-rc2010-10-07/inc/common.php~	2010-10-07 21:27:15.000000000 +0300
+++ dokuwiki-rc2010-10-07/inc/common.php	2010-10-07 21:28:01.732549375 +0300
@@ -1073,6 +1073,7 @@
     // decide if there is something to do
     if($who == 'admin'){
         if(empty($conf['notify'])) return; //notify enabled?
+        if($conf['useacl'] && $_SERVER['REMOTE_USER'] && $minor) return; //skip minors
         $text = rawLocale('mailtext');
         $to   = $conf['notify'];
         $bcc  = '';
