--- dokuwiki-2007-06-26b/inc/parserutils.php	2007-06-26 21:27:15.000000000 +0300
+++ dokuwiki/inc/parserutils.php	2007-11-29 11:47:57.797811307 +0200
@@ -585,16 +585,16 @@
   $cache = getCacheName($language.$code,".code");
 
   if (@file_exists($cache) && !$_REQUEST['purge'] &&
-     (filemtime($cache) > filemtime(DOKU_INC . 'inc/geshi.php'))) {
+     (filemtime($cache) > filemtime('/usr/share/php/geshi.php'))) {
 
     $highlighted_code = io_readFile($cache, false);
     @touch($cache);
 
   } else {
 
-    require_once(DOKU_INC . 'inc/geshi.php');
+    require_once '/usr/share/php/geshi.php';
 
-    $geshi = new GeSHi($code, strtolower($language), DOKU_INC . 'inc/geshi');
+    $geshi = new GeSHi($code, strtolower($language));
     $geshi->set_encoding('utf-8');
     $geshi->enable_classes();
     $geshi->set_header_type(GESHI_HEADER_PRE);
