--- dokuwiki-2013-02-01/inc/common.php~	2013-01-28 00:59:02.000000000 +0200
+++ dokuwiki-2013-02-01/inc/common.php	2013-02-01 01:29:08.204212401 +0200
@@ -1110,6 +1110,7 @@
     // decide if there is something to do, eg. whom to mail
     if($who == 'admin') {
         if(empty($conf['notify'])) return false; //notify enabled?
+        if($conf['useacl'] && $_SERVER['REMOTE_USER'] && $minor) return; //skip minors
         $tpl = 'mailtext';
         $to  = $conf['notify'];
     } elseif($who == 'subscribers') {
