CHECKME: possibly outdated now?

--- dokuwiki-rc2010-10-07/inc/media.php~	2010-10-07 21:28:44.000000000 +0300
+++ dokuwiki-rc2010-10-07/inc/media.php	2010-10-07 21:30:12.262536013 +0300
@@ -392,6 +392,7 @@
     global $lang;
     global $conf;
     global $INFO;
+    global $USERINFO;
     if(empty($conf['notify'])) return; //notify enabled?
 
     $ip = clientIP();
@@ -316,8 +317,8 @@
 
     $from = $conf['mailfrom'];
     $from = str_replace('@USER@',$_SERVER['REMOTE_USER'],$from);
-    $from = str_replace('@NAME@',$INFO['userinfo']['name'],$from);
-    $from = str_replace('@MAIL@',$INFO['userinfo']['mail'],$from);
+    $from = str_replace('@NAME@',$USERINFO['name'],$from);
+    $from = str_replace('@MAIL@',$USERINFO['mail'],$from);
 
     $subject = '['.$conf['title'].'] '.$lang['mail_upload'].' '.$id;
 
