--- dokuwiki-2010-11-07/inc/common.php~	2010-11-07 22:38:45.000000000 +0200
+++ dokuwiki-2010-11-07/inc/common.php	2010-11-07 22:41:54.567137751 +0200
@@ -1163,7 +1163,20 @@
         $subject = '['.utf8_substr($conf['title'], 0, 20).'...] '.$subject;
     }
 
-    mail_send($to,$subject,$text,$conf['mailfrom'],'',$bcc);
+    $headers = array();
+    // add mail headers to identify page changes in same mail thread
+    $file = wikiFN($id);
+    $cur = @filemtime($file);
+    $headers['Message-Id'] = "$id?rev=$cur";
+    if ($rev) {
+        $headers['In-Reply-To'] = "$id?rev=$rev";
+    }
+    $h = '';
+    foreach ($headers as $k => $v) {
+        $h .= sprintf("%s: %s%s", $k, $v, MAILHEADER_EOL);
+    }
+
+    mail_send($to,$subject,$text,$conf['mailfrom'],'',$bcc,$h);
 }
 
 /**
