--- dokuwiki-2012-06-24/conf/dokuwiki.php	2012-07-07 15:08:02.939975138 +0300
+++ dokuwiki-2012-09-23/conf/dokuwiki.php	2012-10-13 18:12:31.417612300 +0300
@@ -19,8 +19,9 @@
 $conf['template']    = 'dokuwiki';         //see lib/tpl directory
 $conf['tagline']     = '';                //tagline in header (if template supports it)
 $conf['sidebar']     = 'sidebar';         //name of sidebar in root namespace (if template supports it)
-$conf['license']     = 'cc-by-nc-sa';     //see conf/license.php
-$conf['savedir']     = './data';          //where to store all the files
+$conf['license']     = 'publicdomain';    //see conf/license.php
+$conf['savedir']     = '/var/lib/dokuwiki';  //where to store all the files
+$conf['cachedir']    = '/var/cache/dokuwiki';  //where cache files are stored
 $conf['basedir']     = '';                //absolute dir from serveroot - blank for autodetection
 $conf['baseurl']     = '';                //URL to server including protocol - blank for autodetect
 $conf['cookiedir']   = '';                //path to use in cookies - blank for basedir
@@ -139,7 +140,7 @@
 $conf['rss_show_summary'] = 1;           //Add revision summary to title? 0|1
 
 /* Advanced Settings */
-$conf['updatecheck'] = 1;                //automatically check for new releases?
+$conf['updatecheck'] = 0;                //automatically check for new releases?
 $conf['userewrite']  = 0;                //this makes nice URLs: 0: off 1: .htaccess 2: internal
 $conf['useslash']    = 0;                //use slash instead of colon? only when rewrite is on
 $conf['sepchar']     = '_';              //word separator character in page names; may be a
--- dokuwiki-2014-05-06/install.php~	2014-05-08 11:32:46.919361482 +0300
+++ dokuwiki-2014-05-06/install.php	2014-05-08 11:33:23.585438490 +0300
@@ -59,7 +59,7 @@
     '2012-09-10'   => 'eb0b3fc90056fbc12bac6f49f7764df3',
     '2013-05-10'   => '7b62b75245f57f122d3e0f8ed7989623',
     '2013-12-08'   => '263c76af309fbf083867c18a34ff5214',
-    '2014-05-05'   => '263c76af309fbf083867c18a34ff5214',
+    '2014-05-05'   => '5527d61a24b4feb0ade5a952848108fb',
 );
 
 
