--- dokuwiki-2012-01-25/conf/dokuwiki.php~	2012-01-27 09:33:02.000000000 +0200
+++ dokuwiki-2012-01-25/conf/dokuwiki.php	2012-01-27 09:33:34.363230629 +0200
@@ -19,7 +19,7 @@
 $conf['basedir']     = '';                //absolute dir from serveroot - blank for autodetection
 $conf['baseurl']     = '';                //URL to server including protocol - blank for autodetect
 $conf['cookiedir']   = '';                //Cookie path. Leave blank for using baseurl.
-$conf['savedir']     = './data';          //where to store all the files
+$conf['savedir']     = '/var/lib/dokuwiki';  //where to store all the files
 $conf['allowdebug']  = 0;                 //allow debug output, enable if needed 0|1
 $conf['mediarevisions'] = 1;              //enable/disable media revisions
 
@@ -26,7 +26,7 @@
 $conf['start']       = 'start';           //name of start page
 $conf['title']       = 'PLD Linux DokuWiki'; //what to show in the title
 $conf['template']    = 'default';         //see lib/tpl directory
-$conf['license']     = 'cc-by-nc-sa';     //see conf/license.php
+$conf['license']     = 'publicdomain';    //see conf/license.php
 $conf['fullpath']    = 0;                 //show full path of the document or relative to datadir only? 0|1
 $conf['recent']      = 20;                //how many entries to show in recent
 $conf['breadcrumbs'] = 10;                //how many recent visited pages to show
--- dokuwiki-2012-01-25/install.php~	2012-01-27 09:34:09.000000000 +0200
+++ dokuwiki-2012-01-25/install.php	2012-01-27 09:34:34.872417423 +0200
@@ -50,7 +50,7 @@
     '2010-11-07'   => '7921d48195f4db21b8ead6d9bea801b8',
     '2011-05-25'   => '4241865472edb6fa14a1227721008072',
     '2011-11-10'   => 'b46ff19a7587966ac4df61cbab1b8b31',
-    '2012-01-25'   => '72c083c73608fc43c586901fd5dabb74',
+    '2012-01-25pld' => '9e3d022699048b720ff54b6bdf49118f',
 );
 
 
