--- dokuwiki-rc2010-10-07/inc/common.php~	2010-10-07 21:34:10.000000000 +0300
+++ dokuwiki-rc2010-10-07/inc/common.php	2010-10-07 21:36:02.099167786 +0300
@@ -1168,7 +1168,20 @@
     $from = str_replace('@NAME@',$INFO['userinfo']['name'],$from);
     $from = str_replace('@MAIL@',$INFO['userinfo']['mail'],$from);
 
-    mail_send($to,$subject,$text,$from,'',$bcc);
+    $headers = array();
+    // add mail headers to identify page changes in same mail thread
+    $file = wikiFN($id);
+    $cur = @filemtime($file);
+    $headers['Message-Id'] = "$id?rev=$cur";
+    if ($rev) {
+        $headers['In-Reply-To'] = "$id?rev=$rev";
+    }
+    $h = '';
+    foreach ($headers as $k => $v) {
+        $h .= sprintf("%s: %s%s", $k, $v, MAILHEADER_EOL);
+    }
+
+    mail_send($to,$subject,$text,$from,'' ,$bcc, $h);
 }
 
 /**
