- https://github.com/dokuwiki/dokuwiki/pull/15

--- dokuwiki-2020-07-29a/conf/dokuwiki.php~	2022-09-03 19:58:24.000000000 +0300
+++ dokuwiki-2020-07-29a/conf/dokuwiki.php	2023-03-27 11:25:04.212540600 +0300
@@ -63,6 +63,7 @@
 $conf['rememberme'] = 1;                 //Enable/disable remember me on login
 $conf['disableactions'] = '';            //comma separated list of actions to disable
 $conf['auth_security_timeout'] = 900;    //time (seconds) auth data is considered valid, set to 0 to recheck on every page view
+$conf['http_auth'] = 1;                  //allows HTTP authorization (SSO) from PHP_AUTH_USER/PHP_AUTH_PW variables
 $conf['securecookie'] = 1;               //never send HTTPS cookies via HTTP
 $conf['remote']      = 0;                //Enable/disable remote interfaces
 $conf['remoteuser']  = '!!not set!!';    //user/groups that have access to remote interface (comma separated). leave empty to allow all users
--- dokuwiki-2018-04-22c.org/inc/auth.php	2020-05-01 12:38:20.000000000 +0200
+++ dokuwiki-2018-04-22c/inc/auth.php	2022-09-27 22:46:31.557879706 +0200
@@ -84,7 +84,7 @@ function auth_setup() {
     }
 
     // if no credentials were given try to use HTTP auth (for SSO)
-    if(!$INPUT->str('u') && empty($_COOKIE[DOKU_COOKIE]) && !empty($_SERVER['PHP_AUTH_USER'])) {
+    if($conf['http_auth'] && !$INPUT->str('u') && empty($_COOKIE[DOKU_COOKIE]) && !empty($_SERVER['PHP_AUTH_USER'])) {
         $INPUT->set('u', $_SERVER['PHP_AUTH_USER']);
         $INPUT->set('p', $_SERVER['PHP_AUTH_PW']);
         $INPUT->set('http_credentials', true);
