--- dokuwiki-rc2009-02-06.new/conf/dokuwiki.php	2009-02-07 10:53:46.339771628 +0100
+++ dokuwiki-0.0.20090214/conf/dokuwiki.php	2009-02-14 19:18:54.000000000 +0100
@@ -18,7 +18,7 @@
 $conf['lang']        = 'en';              //your language
 $conf['basedir']     = '';                //absolute dir from serveroot - blank for autodetection
 $conf['baseurl']     = '';                //URL to server including protocol - blank for autodetect
-$conf['savedir']     = './data';          //where to store all the files
+$conf['savedir']     = '/var/lib/dokuwiki';          //where to store all the files
 $conf['allowdebug']  = 0;                 //allow debug output, enable if needed 0|1
 
 /* Display Options */
@@ -26,7 +26,7 @@
 $conf['start']       = 'start';           //name of start page
 $conf['title']       = 'DokuWiki';        //what to show in the title
 $conf['template']    = 'default';         //see lib/tpl directory
-$conf['license']     = 'cc-by-nc-sa';     //see conf/license.php
+$conf['license']     = 'publicdomain';    //see conf/license.php
 $conf['fullpath']    = 0;                 //show full path of the document or relative to datadir only? 0|1
 $conf['recent']      = 20;                //how many entries to show in recent
 $conf['breadcrumbs'] = 10;                //how many recent visited pages to show
--- dokuwiki-rc2009-12-02/install.php~	2009-12-07 20:47:01.000000000 +0200
+++ dokuwiki-rc2009-12-02/install.php	2009-12-07 20:47:41.702560038 +0200
@@ -47,6 +47,7 @@
     '2008-05-04'   => '1e5c42eac3219d9e21927c39e3240aad',
     '2009-02-14'   => 'ec8c04210732a14fdfce0f7f6eead865',
     'rc2009-12-02' => '993c4b2b385643efe5abf8e7010e11f4',
+    'rc2009-12-02-pld' => 'b9f4419b5b1b9b915c35f4cd3c66bda1',
 );
 
 
