--- dokuwiki/inc/init.php~	2009-11-10 15:19:36.000000000 +0200
+++ dokuwiki/inc/init.php	2009-11-10 15:19:38.080947917 +0200
@@ -21,7 +21,10 @@
 if(!defined('DOKU_INC')) define('DOKU_INC',fullpath(dirname(__FILE__).'/../').'/');
 
 // define config path (packagers may want to change this to /etc/dokuwiki/)
-if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
+if(!defined('DOKU_CONF')) define('DOKU_CONF','/etc/webapps/dokuwiki/');
+
+// define path where variable data is saved (packagers may want to change this to /var/lib/dokuwiki/)
+if(!defined('DOKU_DATA')) define('DOKU_DATA','/var/lib/dokuwiki/');
 
 // check for error reporting override or set error reporting to sane values
 if (!defined('DOKU_E_LEVEL') && @file_exists(DOKU_CONF.'report_e_all')) {
--- dokuwiki-2012-01-25/install.php~	2012-01-27 09:30:05.000000000 +0200
+++ dokuwiki-2012-01-25/install.php	2012-01-27 09:31:55.795792191 +0200
@@ -6,8 +6,9 @@
  */
 
 if(!defined('DOKU_INC')) define('DOKU_INC',dirname(__FILE__).'/');
-if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
-if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_INC.'conf/');
+if(!defined('DOKU_CONF')) define('DOKU_CONF','/etc/webapps/dokuwiki/');
+if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_CONF);
+if(!defined('DOKU_DATA')) define('DOKU_DATA','/var/lib/dokuwiki/');
 
 // check for error reporting override or set error reporting to sane values
 if (!defined('DOKU_E_LEVEL')) { error_reporting(E_ALL ^ E_NOTICE); }
@@ -422,17 +422,17 @@
 
     $dirs = array(
         'conf'        => DOKU_LOCAL,
-        'data'        => DOKU_INC.'data',
-        'pages'       => DOKU_INC.'data/pages',
-        'attic'       => DOKU_INC.'data/attic',
-        'media'       => DOKU_INC.'data/media',
-        'media_attic' => DOKU_INC.'data/media_attic',
-        'media_meta'  => DOKU_INC.'data/media_meta',
-        'meta'        => DOKU_INC.'data/meta',
-        'cache'       => DOKU_INC.'data/cache',
-        'locks'       => DOKU_INC.'data/locks',
-        'index'       => DOKU_INC.'data/index',
-        'tmp'         => DOKU_INC.'data/tmp'
+        'data'        => DOKU_DATA,
+        'pages'       => DOKU_DATA.'pages',
+        'attic'       => DOKU_DATA.'attic',
+        'media'       => DOKU_DATA.'media',
+        'media_attic' => DOKU_DATA.'media_attic',
+        'media_meta'  => DOKU_DATA.'media_meta',
+        'meta'        => DOKU_DATA.'meta',
+        'cache'       => DOKU_DATA.'cache',
+        'locks'       => DOKU_DATA.'locks',
+        'index'       => DOKU_DATA.'index',
+        'tmp'         => DOKU_DATA.'tmp'
     );
 
     $ok = true;
