--- dokuwiki-2012-07-16/inc/common.php~	2012-07-16 00:15:28.000000000 +0300
+++ dokuwiki-2012-07-16/inc/common.php	2012-07-16 00:20:31.431313996 +0300
@@ -1210,6 +1210,15 @@
             false
         );
     }
+
+    // add mail headers to identify page changes in same mail thread
+    $file = wikiFN($id);
+    $cur = @filemtime($file);
+    $mail->setHeader('Message-Id', "$id?rev=$cur", false);
+    if ($rev) {
+        $mail->setHeader('In-Reply-To', "$id?rev=$rev", false);
+    }
+
     return $mail->send();
 }
 
