--- dokuwiki/lib/plugins/config/settings/config.class.php	2007-06-26 21:27:15.000000000 +0300
+++ dokuwiki/lib/plugins/config/settings/config.class.php	2007-11-20 01:42:59.722673768 +0200
@@ -138,7 +138,7 @@
         }else{
             $contents = '';
         }
-        $pattern = '/\$'.$this->_name.'\[[\'"]([^=]+)[\'"]\] ?= ?(.*?);(?=[^;]*(?:\$'.$this->_name.'|@include|$))/s';
+        $pattern = '/\$'.$this->_name.'\[[\'"]([^=]+)[\'"]\] ?= ?(.*?);(?=[^;]*(?:\$'.$this->_name.'|@?(?:require|include)(?:_once)?|$))/s';
         $matches=array();
         preg_match_all($pattern,$contents,$matches,PREG_SET_ORDER);
 
