--- dokuwiki-2015-08-10/vendor/composer/autoload_classmap.php~	2015-08-10 11:06:28.000000000 +0300
+++ dokuwiki-2015-08-10/vendor/composer/autoload_classmap.php	2015-08-11 15:17:48.771838877 +0300
@@ -6,5 +6,5 @@
 $baseDir = dirname($vendorDir);
 
 return array(
-    'GeSHi' => $vendorDir . '/easybook/geshi/geshi.php',
+    'GeSHi' => '/usr/share/php/geshi.php',
 );
--- dokuwiki-2017-02-19c/vendor/composer/autoload_static.php~	2017-08-11 17:32:50.000000000 +0300
+++ dokuwiki-2017-02-19c/vendor/composer/autoload_static.php	2017-08-18 09:01:48.034574467 +0300
@@ -44,7 +44,7 @@
     );
 
     public static $classMap = array (
-        'GeSHi' => __DIR__ . '/..' . '/easybook/geshi/geshi.php',
+        'GeSHi' => '/usr/share/php/geshi.php',
     );
 
     public static function getInitializer(ClassLoader $loader)
