--- dokuwiki-rc2009-02-06/conf/dokuwiki.php	2009-02-06 09:10:44.000000000 +0100
+++ dokuwiki-rc2009-02-06.new/conf/dokuwiki.php	2009-02-07 10:53:46.339771628 +0100
@@ -18,7 +18,7 @@
 $conf['lang']        = 'en';              //your language
 $conf['basedir']     = '';                //absolute dir from serveroot - blank for autodetection
 $conf['baseurl']     = '';                //URL to server including protocol - blank for autodetect
-$conf['savedir']     = './data';          //where to store all the files
+$conf['savedir']     = '/var/lib/dokuwiki';          //where to store all the files
 $conf['allowdebug']  = 0;                 //allow debug output, enable if needed 0|1
 
 /* Display Options */
--- dokuwiki/install.php~	2009-11-10 15:24:36.000000000 +0200
+++ dokuwiki/install.php	2009-11-10 15:25:06.540902430 +0200
@@ -46,6 +46,7 @@
     '2007-06-26'   => 'b3ca19c7a654823144119980be73cd77',
     '2008-05-04'   => '1e5c42eac3219d9e21927c39e3240aad',
     '2009-02-14'   => 'ec8c04210732a14fdfce0f7f6eead865',
+    '2009-02-14-pld'   => 'f2ea17d864c3d7995418e58b85dbb8d0',
 );
 
 
