--- dokuwiki-2009-02-14/install.php~	2009-11-10 07:54:33.847106003 +0200
+++ dokuwiki-2009-02-14/install.php	2009-11-10 07:54:11.370025985 +0200
@@ -373,7 +373,7 @@
 
     // configs shouldn't exist
     foreach ($config_files as $file) {
-        if (@file_exists($file)) {
+        if (@file_exists($file) && filesize($file)) {
             $file    = str_replace($_SERVER['DOCUMENT_ROOT'],'{DOCUMENT_ROOT}/', $file);
             $error[] = sprintf($lang['i_confexists'],$file);
             $ok      = false;
