--- dokuwiki-2014-05-06/inc/parser/xhtml.php~	2014-05-08 11:24:44.093792329 +0300
+++ dokuwiki-2014-05-06/inc/parser/xhtml.php	2014-05-08 11:25:15.469954476 +0300
@@ -689,7 +689,7 @@
 
     /**
      */
-    function interwikilink($match, $name = null, $wikiName, $wikiUri) {
+    function interwikilink($match, $name = null, $wikiName, $wikiUri, $returnonly = false) {
         global $conf;
 
         $link = array();
@@ -598,7 +598,11 @@
         $link['title'] = htmlspecialchars($link['url']);
 
         //output formatted
-        $this->doc .= $this->_formatLink($link);
+        if($returnonly){
+            return $this->_formatLink($link);
+        }else{
+            $this->doc .= $this->_formatLink($link);
+        }
     }
 
     /**
