--- dokuwiki-2009-02-14/inc/parser/xhtml.php	2009-07-28 16:51:06.599541994 +0300
+++ dokuwiki/inc/parser/xhtml.php	2009-07-28 16:53:48.144936039 +0300
@@ -569,7 +569,7 @@
 
     /**
     */
-    function interwikilink($match, $name = NULL, $wikiName, $wikiUri) {
+    function interwikilink($match, $name = NULL, $wikiName, $wikiUri, $returnonly = false) {
         global $conf;
 
         $link = array();
@@ -598,7 +598,11 @@
         $link['title'] = htmlspecialchars($link['url']);
 
         //output formatted
-        $this->doc .= $this->_formatLink($link);
+        if($returnonly){
+            return $this->_formatLink($link);
+        }else{
+            $this->doc .= $this->_formatLink($link);
+        }
     }
 
     /**
