--- dokuwiki-rc2013-10-28/inc/parser/xhtml.php~	2013-11-13 22:20:29.920364255 +0200
+++ dokuwiki-rc2013-10-28/inc/parser/xhtml.php	2013-11-13 22:20:52.460009959 +0200
@@ -686,7 +686,7 @@
 
     /**
     */
-    function interwikilink($match, $name = null, $wikiName, $wikiUri) {
+    function interwikilink($match, $name = null, $wikiName, $wikiUri, $returnonly = false) {
         global $conf;
 
         $link = array();
@@ -598,7 +598,11 @@
         $link['title'] = htmlspecialchars($link['url']);
 
         //output formatted
-        $this->doc .= $this->_formatLink($link);
+        if($returnonly){
+            return $this->_formatLink($link);
+        }else{
+            $this->doc .= $this->_formatLink($link);
+        }
     }
 
     /**
