--- sidebar/main.php    2009-08-29 23:53:54.718210429 +0300
+++ sidebar/main.php    2009-09-01 12:51:33.000000000 +0300
@@ -18,6 +18,8 @@
 // must be run from within DokuWiki
 if (!defined('DOKU_INC')) die();
 
+global $REV;
+
 // include functions that provide sidebar functionality
 @require_once(dirname(__FILE__).'/tplfn_sidebar.php');
 ?>
@@ -66,6 +68,30 @@
       </div>
 
       <div class="bar-right" id="bar__topright">
+        <?php
+        $pdf_plugin = &plugin_load('action','html2pdf');
+        if ($pdf_plugin && !plugin_isdisabled($pdf_plugin->getPluginName())) { ?>
+        <form class="button" method="get" action="<?php wl($ID)?>">
+          <div class="no">
+            <input type="submit" value="Export to PDF" class="button" />
+            <input type="hidden" name="do" value="export_pdf" />
+            <input type="hidden" name="id" value="<?php echo $ID?>" />
+            <input type="hidden" name="rev" value="<?php echo $REV ?>" />
+          </div>
+        </form>
+        <?php } ?>
+        <?php
+        $odt_plugin = &plugin_load('renderer','odt');
+        if ($odt_plugin && !plugin_isdisabled($odt_plugin->getPluginName())) { ?>
+        <form class="button" method="get" action="<?php wl($ID)?>">
+          <div class="no">
+            <input type="submit" value="Export to ODT" class="button" />
+            <input type="hidden" name="do" value="export_odt" />
+            <input type="hidden" name="id" value="<?php echo $ID?>" />
+            <input type="hidden" name="rev" value="<?php echo $REV ?>" />
+          </div>
+        </form>
+        <?php } ?>
         <?php tpl_button('recent')?>
         <?php tpl_searchform()?>&nbsp;
       </div>
