--- dokuwiki-rc2011-11-10/lib/tpl/default/main.php~	2012-02-11 15:16:44.000000000 +0200
+++ dokuwiki-rc2011-11-10/lib/tpl/default/main.php	2012-02-11 15:17:57.971423003 +0200
@@ -15,6 +15,8 @@
 // must be run from within DokuWiki
 if (!defined('DOKU_INC')) die();
 
+global $REV;
+
 ?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
@@ -66,6 +68,28 @@
       </div>
 
       <div class="bar-right" id="bar__topright">
+        <?php
+        if (!plugin_isdisabled('html2pdf')) { ?>
+        <form class="button" method="get" action="<?php wl($ID)?>">
+          <div class="no">
+            <input type="submit" value="Export to PDF" class="button" />
+            <input type="hidden" name="do" value="export_pdf" />
+            <input type="hidden" name="id" value="<?php echo $ID?>" />
+            <input type="hidden" name="rev" value="<?php echo $REV ?>" />
+          </div>
+        </form>
+        <?php } ?>
+        <?php
+        if (!plugin_isdisabled('odt')) { ?>
+        <form class="button" method="get" action="<?php wl($ID)?>">
+          <div class="no">
+            <input type="submit" value="Export to ODT" class="button" />
+            <input type="hidden" name="do" value="export_odt" />
+            <input type="hidden" name="id" value="<?php echo $ID?>" />
+            <input type="hidden" name="rev" value="<?php echo $REV ?>" />
+          </div>
+        </form>
+        <?php } ?>
         <?php tpl_button('recent')?>
         <?php tpl_searchform()?>&nbsp;
       </div>
