--- dokuwiki-2015-08-10/lib/tpl/dokuwiki/tpl_header.php~	2015-08-11 15:39:31.799253674 +0300
+++ dokuwiki-2015-08-10/lib/tpl/dokuwiki/tpl_header.php	2015-08-11 15:39:49.525658460 +0300
@@ -66,6 +66,19 @@
             </div>
             <ul>
                 <?php
+                    global $REV;
+                    if (!plugin_isdisabled('html2pdf') || !plugin_isdisabled('dw2pdf')) {
+                        echo '<li>';
+                        tpl_link(wl($ID, array('do' => 'export_pdf', 'rev' => $REV)), '<span>PDF Export</span>', 'class="action html2pdf"');
+                        echo '</li>';
+                    }
+
+                    if (!plugin_isdisabled('odt')) {
+                        echo '<li>';
+                        tpl_link(wl($ID, array('do' => 'export_odt', 'rev' => $REV)), '<span>ODT Export</span>', 'class="action odt"');
+                        echo '</li>';
+                    }
+
                     tpl_toolsevent('sitetools', array(
                         tpl_action('recent', true, 'li', true),
                         tpl_action('media', true, 'li', true),
