--- dokuwiki-2009-02-14/inc/plugincontroller.class.php~	2009-02-14 14:13:25.000000000 +0200
+++ dokuwiki-2009-02-14/inc/plugincontroller.class.php	2009-10-13 04:18:14.000000000 +0300
@@ -79,7 +79,7 @@
     $dir = $this->get_directory($plugin);
     $file = $component ? "$type/$component.php" : "$type.php";
 
-    if (!include_once(DOKU_PLUGIN."$dir/$file")) {
+    if (!file_exists(DOKU_PLUGIN."$dir/$file") || !include_once(DOKU_PLUGIN."$dir/$file")) {
       return null;
     }
 
