--- dokuwiki/lib/scripts/script.js~	2010-04-29 15:42:12.000000000 +0300
+++ dokuwiki/lib/scripts/script.js	2010-04-29 15:43:39.640053591 +0300
@@ -512,7 +512,7 @@
         for(var i=0; i<elems.length; i++){
             var share = elems[i];
             addEvent(share,'click',function(){
-                if(document.all == null) {
+                if(document.all == null && LANG['nosmblinks']) {
                     alert(LANG['nosmblinks']);
                 }
             });
