--- dokuwiki-2013-03-05/lib/exe/js.php~	2013-03-03 21:16:43.000000000 +0200
+++ dokuwiki-2013-03-05/lib/exe/js.php	2013-03-05 12:40:14.025436522 +0200
@@ -41,8 +41,8 @@
     // array of core files
     $files = array(
                 DOKU_INC."lib/scripts/jquery/jquery$min.js",
-                DOKU_INC.'lib/scripts/jquery/jquery.cookie.js',
-                DOKU_INC."lib/scripts/jquery/jquery-ui$min.js",
+                '/usr/share/jquery/cookie/cookie.js',
+                '/usr/share/jquery/ui/jquery-ui.js',
                 DOKU_INC."lib/scripts/jquery/jquery-migrate$min.js",
                 DOKU_INC."lib/scripts/fileuploader.js",
                 DOKU_INC."lib/scripts/fileuploaderextended.js",
