--- dokuwiki-rc2013-10-28/inc/load.php~	2013-11-13 23:16:52.413946070 +0200
+++ dokuwiki-rc2013-10-28/inc/load.php	2013-11-13 23:26:34.714816053 +0200
@@ -82,10 +82,7 @@
         'RemoteAPI'             => DOKU_INC.'inc/remote.php',
         'RemoteAPICore'         => DOKU_INC.'inc/RemoteAPICore.php',
         'Subscription'          => DOKU_INC.'inc/subscription.php',
-        'Crypt_Base'            => DOKU_INC.'inc/phpseclib/Crypt_Base.php',
-        'Crypt_Rijndael'        => DOKU_INC.'inc/phpseclib/Crypt_Rijndael.php',
-        'Crypt_AES'             => DOKU_INC.'inc/phpseclib/Crypt_AES.php',
-        'Crypt_Hash'            => DOKU_INC.'inc/phpseclib/Crypt_Hash.php',
+        'Crypt_AES'             => '/usr/share/pear/Crypt/AES.php',
         'lessc'                 => DOKU_INC.'inc/lessc.inc.php',
 
         'DokuWiki_Action_Plugin' => DOKU_PLUGIN.'action.php',
