--- dokuwiki-rc2013-10-28/inc/load.php~	2013-11-13 23:27:34.370547242 +0200
+++ dokuwiki-rc2013-10-28/inc/load.php	2013-11-14 20:18:16.051225452 +0200
@@ -83,7 +83,7 @@
         'RemoteAPICore'         => DOKU_INC.'inc/RemoteAPICore.php',
         'Subscription'          => DOKU_INC.'inc/subscription.php',
         'Crypt_AES'             => '/usr/share/pear/Crypt/AES.php',
-        'lessc'                 => DOKU_INC.'inc/lessc.inc.php',
+        'lessc'                 => '/usr/share/php/lessc.inc.php',
 
         'DokuWiki_Action_Plugin' => DOKU_PLUGIN.'action.php',
         'DokuWiki_Admin_Plugin'  => DOKU_PLUGIN.'admin.php',
