--- dokuwiki-2017-02-19b/inc/load.php~	2017-03-28 20:41:05.000000000 +0300
+++ dokuwiki-2017-02-19b/inc/load.php	2017-03-28 20:41:30.364262022 +0300
@@ -83,7 +83,7 @@
         'RemoteAPI'             => DOKU_INC.'inc/remote.php',
         'RemoteAPICore'         => DOKU_INC.'inc/RemoteAPICore.php',
         'Subscription'          => DOKU_INC.'inc/subscription.php',
-        'lessc'                 => DOKU_INC.'inc/lessc.inc.php',
+        'lessc'                 => '/usr/share/php/lessc.inc.php',
 
         'DokuWiki_Action_Plugin' => DOKU_PLUGIN.'action.php',
         'DokuWiki_Admin_Plugin'  => DOKU_PLUGIN.'admin.php',
