--- dokuwiki-2014-05-06/lib/exe/js.php~	2014-05-06 15:09:52.000000000 +0300
+++ dokuwiki-2014-05-06/lib/exe/js.php	2014-05-08 11:22:58.838818098 +0300
@@ -40,9 +40,9 @@
 
     // array of core files
     $files = array(
-                DOKU_INC."lib/scripts/jquery/jquery$min.js",
-                DOKU_INC.'lib/scripts/jquery/jquery.cookie.js',
-                DOKU_INC."lib/scripts/jquery/jquery-ui$min.js",
+                "/usr/share/jquery/jquery$min.js",
+                "/usr/share/jquery/cookie/cookie.js",
+                "/usr/share/jquery/ui/jquery-ui.js",
                 DOKU_INC."lib/scripts/jquery/jquery-migrate$min.js",
                 DOKU_INC.'inc/lang/'.$conf['lang'].'/jquery.ui.datepicker.js',
                 DOKU_INC."lib/scripts/fileuploader.js",
