--- dokuwiki/inc/html.php~	2010-01-11 14:35:04.000000000 +0200
+++ dokuwiki/inc/html.php	2010-01-11 14:35:07.213389784 +0200
@@ -862,8 +862,12 @@
     // given as rev and rev2 parameters, with rev2 being optional. Or in an
     // array in rev2.
     $rev1 = $REV;
+	// handle $r1 + $r2, see DW #1821
+	if (isset($_REQUEST['r1']) && isset($_REQUEST['r2'])) {
+        $rev1 = (int) $_REQUEST['r1'];
+        $rev2 = (int) $_REQUEST['r2'];
 
-    if(is_array($_REQUEST['rev2'])){
+	} else if(is_array($_REQUEST['rev2'])){
         $rev1 = (int) $_REQUEST['rev2'][0];
         $rev2 = (int) $_REQUEST['rev2'][1];
 
--- dokuwiki/inc/html.php~	2010-10-07 21:48:30.000000000 +0300
+++ dokuwiki/inc/html.php	2010-10-18 17:28:03.106448956 +0300
@@ -997,7 +997,7 @@
     if($intro) print p_locale_xhtml('diff');
 
     if (!$text) {
-        $diffurl = wl($ID, array('do'=>'diff', 'rev2[0]'=>$l_rev, 'rev2[1]'=>$r_rev));
+        $diffurl = wl($ID, array('do'=>'diff', 'r1'=>$l_rev, 'r2'=>$r_rev));
         ptln('<p class="difflink">');
         ptln('  <a class="wikilink1" href="'.$diffurl.'">'.$lang['difflink'].'</a>');
         ptln('</p>');
--- dokuwiki-rc2010-10-07/inc/common.php~	2010-10-07 21:41:20.000000000 +0300
+++ dokuwiki-rc2010-10-07/inc/common.php	2010-10-07 21:45:00.639114275 +0300
@@ -1147,6 +1147,8 @@
     }elseif($rev){
         $subject = $lang['mail_changed'].' '.$id;
         $text = str_replace('@OLDPAGE@',wl($id,"rev=$rev",true,'&'),$text);
+        $rev2 = $INFO['meta']['date']['modified'];
+        $text = str_replace('@DIFFPAGE@',wl($id,"do=diff&r1=$rev&r2=$rev2",true,'&'),$text);
         $df  = new Diff(explode("\n",rawWiki($id,$rev)),
                         explode("\n",rawWiki($id)));
         $dformat = new UnifiedDiffFormatter();
@@ -1154,6 +1156,7 @@
     }else{
         $subject=$lang['mail_newpage'].' '.$id;
         $text = str_replace('@OLDPAGE@','none',$text);
+        $text = str_replace('@DIFFPAGE@','none',$text);
         $diff = rawWiki($id);
     }
     $text = str_replace('@DIFF@',$diff,$text);
